.github/build.in.yml

```diff
@@ -207,7 +207,7 @@ jobs:
           # storing and transferring oleans over the network.
           # Hopefully a future re-implementation of `cache` will obviate the present need for this hack.
           lake exe cache get Archive.lean
-          bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build Archive"
+          bash -o pipefail -c "env LEAN_ABORT_ON_PANIC=1 lake build --wfail Archive"
```

Comment:
I think that this means that CI will stop here, if `Archive` does not build and will not continue on to checking the status of `Counterexamples`.

I would rather it continued, though, and report an error after it tried to build both.

Reply:
Agreed. I pushed one way to fix this --- which is not the most elegant, but works. Up to you: do you think the PR is worth it overall?